index.html

```diff
@@ -0,0 +1,176 @@
+    </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="skils">
+       <li class="button">html</li>
+       <li class="skil button">bootstrap</li>
+       <li class="skil button">Ruby</li>
+    </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="skils">
+       <li class="button">html</li>
+       <li class="skil button">bootstrap</li>
+       <li class="skil button">Ruby</li>
+    </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+    
+</section>
+
+<section class="section-3">
+   <div class="about-me">
+      <h1 class="h31">About me</h1>
+      <p class="p31">Hello I’m a software developer! I can help you build a product ,feature or 
+         website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <a class="get-myresume button" href="">Get My Resume</a>
+      <img class="abt-me-image" src="./assets/images/Group1.png" alt="">
+   </div>
+   <hr class="line-3">
+
+   <ul class="lists">
+      <li class="Lan">Language 
+          <ul class="lan-list">
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>Css</li>
+            </ul>
+      </li>
+        
+      <li class="Frame"> Framework
+          <ul class="Frame-list">
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li>
+          </ul>
+      </li>
+      <li class="Sk">Skills
+          <ul class="Sk-list">
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+      </li>
+  </ul>
+</section>
+        <form class="upper-form" action="https://formspree.io/f/xoqrrprd" method="post">
+         <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch. </p>
+            <input type="text" maxlength="30" required placeholder="Full Name" class="full-name" name="usrname">
+            <input type="email" required placeholder="Email" class="email" name="usremail">
+            <textarea maxlength="500" id="msg" placeholder="Enter text here" class="messages" name="usrmessages"></textarea>
+            <input type="submit" value="Get it touch" class="touch" name="usrsubmit">
```

Comment:
[OPTIONAL] I think it would be better to add a pressed state to our button. This will increase accessibility of your page 👌
<img width="401" alt="Screenshot 2023-01-25 at 11 18 01 PM" src="https://user-images.githubusercontent.com/25887253/214682300-ed12f34c-dce0-4108-a134-461268d41e97.png">
<img width="401" alt="Screenshot 2023-01-25 at 11 18 55 PM" src="https://user-images.githubusercontent.com/25887253/214682310-404621e2-3eca-493a-b641-1d727cf12d3b.png">